docs/sensors/triggers/argo-workflow.md

```diff
@@ -68,6 +68,6 @@ provided by the Argo CLI such as,
 To make use of Argo CLI operations, The sensor provides the `argoWorkflow` trigger template,
 
         argoWorkflow:
-          operation: submit  # submit, resubmit, resume, retry, suspend, terminate or stop
+          operation: submit  # submit, submit-from, resubmit, resume, retry, suspend, terminate or stop
```

Comment:
Line 62 - 66 as well, also, appreciate if you could add the other missing operations there (terminate, stop...).

Reply:
updated!